app/javascript/chat/chat.jsx

```diff
@@ -1781,6 +1781,7 @@ export default class Chat extends Component {
             : 'message__delete__modal message__delete__modal__hide crayons-modal crayons-modal--s absolute'
         }
         aria-hidden={showDeleteModal}
+        aria-label="delete confirmation"
```

Comment:
@maestromac, looks like this is a bad merge?

Reply:
Nope, this PR also happens to bump jest-axe from 4.1.0 to 4.1.1 :neutral_face: . Jest was failing so I tried my best to fix it. Did I do it wrong?

Reply:
I'm just curious why we had to add this now. I would've assumed this would have been previously failing. Just gonna pull this locally.

Reply:
It was breaking this rule https://dequeuniversity.com/rules/axe/4.1/aria-dialog-name?application=axeAPI